torch/fx/experimental/symbolic_shapes.py

```diff
@@ -177,6 +178,26 @@ def eval(cls, base, divisor):
                     sympy.simplify(base / gcd), sympy.simplify(divisor / gcd)
                 )
 
+    class Ceil(sympy.Function):
```

Comment:
Why can't we use this here? https://docs.sympy.org/latest/modules/functions/elementary.html

Reply:
^- @bdhirsh

Reply:
Oh yeah we should use ‘ceiling’, I must have missed it. I mostly just copied off of how your did floor div